pkg/kconfig/kconfig.go

```
@@ -0,0 +1,190 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kconfig
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const (
+	ContainerCgroupPath = "/proc/1/cgroup"
+	StackUnwinder       = "CONFIG_UNWINDER_FRAME_POINTER"
+)
+
+// TODO(vthakkar): Add more locations for scanning the config
+// https://github.com/libbpf/libbpf/blob/9a3a42608dbe3731256a5682a125ac1e23bced8f/src/btf.c#L3114-L3122
+var (
+	configPaths = []string{
+		"/proc/config.gz",
+		"/boot/config",
+		"/boot/config-%s",
+	}
+)
```

```suggestion
var configPaths = []string{
		"/proc/config.gz",
		"/boot/config",
		"/boot/config-%s",
	}
```

nit. Or could this var scope could include the below var